synapse/cmds/hive.py

```diff
@@ -144,17 +144,26 @@ def parsepath(path):
             return
 
         if opts.json:
-            rend = json.dumps(valu, indent=2)
+            prend = json.dumps(valu, indent=2)
```

Comment:
adding sort_keys=True would allow for easier diffing of hive entries by hand

Reply:
good idea.